src/marine/glider2ioda.py

```diff
@@ -132,20 +132,21 @@ def main():
     yamlfile = args.yaml
     fdate = datetime.strptime(args.date, '%Y%m%d%H')
     VarDims = {
-        'sea_water_temperature': ['nlocs'],
-        'sea_water_salinity': ['nlocs']}
+        'waterTemperature': ['Location'],
+        'salinity': ['Location']}
     prof = Profile(args.input, args.yaml, fdate)
-    GlobalAttrs['date_time_string'] = fdate.strftime("%Y-%m-%dT%H:%M:%SZ")
     ObsVars, nlocs = iconv.ExtractObsData(prof.data, locationKeyList)
-    DimDict = {'nlocs': nlocs}
+    DimDict = {'Location': nlocs}
     writer = iconv.IodaWriter(args.output, locationKeyList, DimDict)
     VarAttrs = DefaultOrderedDict(lambda: DefaultOrderedDict(dict))
-    VarAttrs[('sea_water_temperature', 'ObsValue')]['units'] = 'celsius'
-    VarAttrs[('sea_water_salinity', 'ObsValue')]['units'] = 'psu'
-    VarAttrs[('sea_water_temperature', 'ObsValue')]['_FillValue'] = -32767
-    VarAttrs[('sea_water_salinity', 'ObsValue')]['_FillValue'] = -32767
-    VarAttrs[('sea_water_temperature', 'PreQC')]['units'] = ''
-    VarAttrs[('sea_water_salinity', 'ObsError')]['units'] = ''
+    VarAttrs[('waterTemperature', 'ObsValue')]['units'] = 'K'
+    VarAttrs[('waterTemperature', 'ObsError')]['units'] = 'K'
+    VarAttrs[('salinity', 'ObsValue')]['units'] = 'unitless'
```

Comment:
This line and one below should be a value of one (1), not unitless.  There is no unit for unitless so any variable without units, this attribute should be skipped entirely.

Reply:
Changed as "1".